rmc/utils/constraint.py

```diff
@@ -1447,44 +1430,36 @@ def fix_transcript_start_stop_aas(
         overwrite=overwrite_temp,
     )
 
-    # Get all positions with defined amino acids in context HT
+    # Get largest defined amino acids present in context HT for each
+    # transcript that is missing an amino acid at its CDS start or end
```

Comment:
I think we can just move this to line 1445 (+ replace the current comment there)